api/tests/Feature/Generators/CommunityInterestUserCsvGeneratorTest.php

```diff
@@ -16,7 +16,7 @@
 use function PHPUnit\Framework\assertEquals;
 use function PHPUnit\Framework\assertTrue;
 
-class CommunityInterestUserCsvGeneratorTest extends TestCase
+class CommunityInterestUserExcelGeneratorTest extends TestCase
```

Comment:
If we aren't updating this download as part of this PR maybe undo the name change here.